safe_relay_service/relay/tasks.py

```diff
@@ -218,8 +218,11 @@ def deploy_safes_task(retry: bool=True) -> None:
                         safe_funding.safe_deployed_tx_hash = creation_tx_hash
                         safe_funding.save()
                 except TransactionAlreadyImported:
-                    logger.warning("Safe=%s transaction was already imported by the node")
-                    return
+                    logger.warning("Safe=%s transaction was already imported by the node", safe_address)
+                    safe_funding.safe_deployed_tx_hash = safe_creation.tx_hash
+                    safe_funding.safe_deployed = False
```

Comment:
Do we have to explicitely set it to `False`

shouldn't it just stay what it is?

Reply:
I found some weird cases when safe is deployed but the tx_hash is empty, and wrote this to make sure that the state is what should be and no other instance is writing `safe_deployed`, but as Django writes all the fields of the object, you are right, it's no use